search-sentiments-map/public/scripts/map-script.js

```diff
@@ -25,14 +25,30 @@ let mapStyle = [{
 }];
 let map;
 let infowindow;
+
+// Multiplier for sentiment scores.
+const SCORE_SCALE_MULTIPLIER = 100;
+// The default score assigned to countries with not search results.
+const NO_RESULTS_DEFAULT_SCORE = -500;
+
 /* 
  * Hold the minimum and maximum values of the sentiment scores.
- * The sentiment API returns scores from -1.0 to 1.0.
+ * The sentiment API returns scores from -1.0 to 1.0. Our value is these max
+ * and min scores multiplied by our score multiplier.
  */
+const DATA_MAX = SCORE_SCALE_MULTIPLIER * 1.0;
+const DATA_MIN = SCORE_SCALE_MULTIPLIER  *-1.0;
 
- // TODO(ntarn@): Change this to 100 and -100 when sentiment scores rescaled.
-const DATAMIN = 100.0;
-const DATAMAX = -100.0;
+/**
+ * HSL color codes for country colorings.
+ * @enum {Array}
+ */
+const CountryColorCodes = {
+  GREEN: [114, 80, 39],//[151, 83, 34],
```

Comment:
Add a reason for the comment? If that's the blue you were considering, go ahead and remove it.

```suggestion
  GREEN: [114, 80, 39], //[151, 83, 34],
```